### PR TITLE
labours: set max_people in show_devs

### PR DESCRIPTION
**Pull request by bzz**
Use and log actual `max_people` from CLI arg in `-m dev`.

### CODE REVIEW

**Comment by bzz** [comment]
CI failure seems somewhat un-related

```
$ $GOPATH/bin/hercules --burndown --burndown-files --burndown-people --couples --devs --quiet https://github.com/src-d/hercules | labours -m all -o out --backend Agg --disable-projector
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 583, in _build_master
    ws.require(__requires__)
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 900, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/local/lib/python3.5/dist-packages/pkg_resources/__init__.py", line 791, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (scipy 1.2.2 (/usr/local/lib/python3.5/dist-packages), Requirement.parse('scipy<=1.2.1,>=1.0'), {'lifelines'})
```

File: python/labours/labours.py [hunk 1]
```diff
@@ -1286,12 +1286,11 @@ def show_sentiment_stats(args, name, resample, start_date, data):
...
 
 
-def show_devs(args, name, start_date, end_date, people, days):
+def show_devs(args, name, start_date, end_date, people, days, max_people=50):
     from scipy.signal import convolve, slepian
 
-    max_people = 50
     if len(people) > max_people:
-        print("Picking top 100 developers by commit count")
+        print("Picking top {} developers by commit count".format(max_people))
```
Comment (vmarkovtsev) [inline python/labours/labours.py]:
```suggestion
        print("Picking top %s developers by commit count" % max_people)
```
Following p.5 of https://github.com/src-d/guide/blob/master/engineering/conventions/python.md#code-style

Comment (bzz) [reply python/labours/labours.py]: Thank you for pointing it out! Addressed, history re-written.

**Comment by bzz** [comment]
Rebased on latest master and all feedback addressed. @vmarkovtsev it's  ready for another round!

**Pull request merged**